frontend/src/UserAPI.js

```diff
@@ -11,6 +15,15 @@ const api2 = axios.create({
     baseURL: `http://localhost:8080/api/users/`,
 })
 
+const api3 = axios.create({
+    baseURL: `https://reqres.in/api/`
+})
+
+async function loginUser(credentials) {
+    const result = await api3.post('/login', credentials);
+    return result.data.token;
```

Comment:
Itt nincs különösebben hibakezelés, nem tudom ezen a ponton az mennyire fontos, talán egyeztessünk erről a PM-mel.

Reply:
Ha az authentikációs kérésünk BE oldali válaszára (sikeres / nem sikeres + valami message) vonatkozik a kérdés, akkor azt mondom, hogy illenék hibakezelni. Egyelőre csak minimálisan, de legyen - majd BE összekötés után finomítható